### PR TITLE
change config into dict

### PR DESCRIPTION
**Pull request by neihousaigaai**

### CODE REVIEW

File: game/__init__.py [hunk 1]
```diff
@@ -76,32 +80,13 @@ def generate_roles(interface, ids, names_dict):
...
-                    ["Werewolf", "Seer"  , "Guard"   , "Villager", "Villager", "Villager", "Villager"],
-                    ["Werewolf", "Seer"  , "Guard"   , "Villager", "Villager", "Lycan"   , "Lycan"],
-                    ["Werewolf", "Seer"  , "Guard"   , "Villager", "Lycan"   , "Lycan"   , "Lycan"],
-                    ["Werewolf", "Seer"  , "Guard"   , "Lycan"   , "Lycan"   , "Lycan"   , "Lycan"]
-                ]
+            role_config = config.DEFAULT_COUNT_CONFIG
 
         ids = list(ids)
-        game_role = random.choice([role_list for role_list in role_config if len(role_list)==len(ids)])
+        game_role = random.choice([dict_to_list(role_dict) for role_dict in role_config if sum(role_dict.values())==len(ids)])
```
Comment (nitro2) [inline game/__init__.py:86]: Can we have 2 cases:
```python
if len(ids) <= sum(role_config[-1].values()):  # Last row in role_config file
  # Use above 
else:
  # Append Villager?
```

File: game/__init__.py [hunk 2]
```diff
@@ -68,6 +68,10 @@ def awake(self):
 
     @staticmethod
     def generate_roles(interface, ids, names_dict):
+        def dict_to_list(config, number=0):
+            yield from (name for name in config for _ in range(config[name]))
+            yield from ('Werewolf' if i%4==0 else 'Villager' for i in range(number-sum(config.values())))
```
Comment (nitro2) [inline game/__init__.py:73]: How about Superwolf ?

Comment (Stanley00) [reply game/__init__.py:73]: Please check these lines for how to add Superwolf https://github.com/dnh-bot/dnh-werewolf-bot/pull/64/commits/5da6099e531a6136eaecb7847b1ac52c9b2dfd5c#diff-117426151e93a626f8b46bfdb3a95b3f4a62e5f4dd6e65975a7c50759bf04482R79

**Review by nitro2 — APPROVED** [review]

File: game/__init__.py (hunk 1 above)
Comment (Stanley00) [reply game/__init__.py]: Done in commit 3fffa9650b

**Review by nitro2 — APPROVED** [review]

**Pull request merged**